src/icesat2_tracks/local_modules/JONSWAP_gamma.py

```diff
@@ -184,12 +170,8 @@ def JONSWAP_default(f, X, U, gamma=3.3):
     peak_factor = gamma**delta
 
     return (
-        alpha
-        * g**2.0
-        * w ** (-5.0)
-        * np.exp(-5.0 / 4.0 * (w / wp) ** -4)
-        * peak_factor
-    )  # Hz**-5 m**2 /s**4  = m**2 sec  ?
+        alpha * g**2.0 * w ** (-5.0) * np.exp(-5.0 / 4.0 * (w / wp) ** -4) * peak_factor
+    )
```

Comment:
Same here – this is useful for someone reading the code
```suggestion
    )  # Hz**-5 m**2 /s**4 = m**2 sec ?
```